components/TodoList.js

```diff
@@ -1,37 +1,41 @@
-import  { useState } from 'react';
+import { useState } from 'react';
 
 function TodoList() {
-  const [todos, setTodos] = useState([]);  
-  const [newTodo, setNewTodo] = useState('');  
+  const [todos, setTodos] = useState([]);
+  const [newTodo, setNewTodo] = useState('');
 
-  function handleAddTodo() {
+  function addNewTodo() {
     if (newTodo.trim() !== '') {
-      
       setTodos([...todos, newTodo]);
       setNewTodo('');
     }
   }
 
+  function removeTodo(index) {
+    const newTodos = todos.filter((_, i) => i !== index);
+    setTodos(newTodos);
+  }
```

Comment:
You will need to adjust this function after adding an id to your state. Instead of `index` you will receive an id and then filter according to a match of the id.

Reply:
Thank you, I changed it